src/components/Address/README.md

```diff
@@ -0,0 +1,34 @@
+# Ethereum Address component
```

Comment:
Are you going to separate this out to separate library?

Reply:
I'm in the middle of separating it out right now: https://github.com/ensdomains/react-ens-address